src/apps/loan-list/utils/helpers.ts

```diff
@@ -53,4 +53,26 @@ export const queryMatchesFaust = (query: string | null) => {
   return returnValue;
 };
 
+export const getStackedSearchItems = (
+  view: string,
```

Comment:
I think that view should be a uniontype. If it is a global type it could be moved to: src/core/utils/types

Reply:
I agree